typescript/best-practice-web/test/pages/inventory.page.ts

```diff
@@ -1,7 +1,8 @@
+import { PAGES } from '../e2eConstants';
 import Page from './page';
 
 class InventoryPage extends Page {
-    private _urlPart: string = '/inventory.html';
+    private _urlPart: string = PAGES.INVENTORY;
 
     open():void {
         super.open(this._urlPart);
```

Comment:
@nikolay-advolodkin 

Why put them in a separate constant? You can just do `super.open(PAGES.INVENTORY);`